tests/acceptance/conftest.py

```diff
@@ -60,8 +63,7 @@ def small_app(request):
         init_all_storage_paths()
         init_users_and_permissions()
 
-        migrate('./inspirehep/demosite/data/demo-records-small.xml',
-                wait_for_results=True)
+        #migrate('./inspirehep/demosite/data/demo-records.xml.gz', wait_for_results=True)
```

Comment:
Sure. It means: "I think you made this modification as a temporary measure while you were exploring the problem, and now you left it inside of the commit. I think you should remove it."